src/pages/onboarding/OnboardingPage.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Carousel from "nuka-carousel";
+
+import "./OnboardingPage.css"
+import ArrowToRightIcon from "../../images/path4.svg";
+
+import OnboardingService from '../../services/OnboardingService'
+
+type Props = {};
+
+type State = {
+    currentIndexCarousel: number
+}
+
+type OnboardingStep = {
+    title: string,
+    description: string,
+    imageSrc: string
+}
+
+const OnboardingSteps: OnboardingStep[] = [
+    {
+        title: 'Welcome to ArcWeather',
+        description: 'Your new favorite weather app, with all the features you need to have the best information available.',
+        imageSrc: 'assets/onboarding-image-01.png'
+    },
+    {
+        title: 'Follow different places',
+        description: 'You can easily add as many different locations as you wish.',
+        imageSrc: 'assets/onboarding-image-02.png'
+    },
+    {
+        title: 'Never forget your umbrella',
+        description: 'Accurate weather forecast that you can trust to never get caught in the rain again.',
+        imageSrc: 'assets/onboarding-image-03.png'
+    }
+]
+
+export default class OnboardingPage extends React.Component<Props, State> {
+
+    onboardingService: OnboardingService
+
+    constructor(props: Props) {
+        super(props)
+        this.moveToNextStep = this.moveToNextStep.bind(this)
+        this.finishOnboarding = this.finishOnboarding.bind(this)
+        this.onboardingService = new OnboardingService()
+    }
+
+    state = {
+        currentIndexCarousel: 0
+    }
+
+    moveToNextStep() {
+        if (this.state.currentIndexCarousel < (OnboardingSteps.length - 1)) {
```

Comment:
For the cases like this where we have a complex boolean expression inside an `if`, I'd suggest us to extract the expression inside a function, that would make the code easier for the developer that is reading it.